### DISCUSSION TITLE
How to insert at the end of a note

### DISCUSSION THREAD

**Discussion by willgraham345** [mProjectsCode/obsidian-meta-bind-plugin#385] [Q&A]
I think this is a relatively easy question, but I haven't been able to find a good answer. I'm looking to insert a template at the end of a file. I can't seem to see a variable that corresponds with the length of the file. Any help is appreciated!

i.e.
```meta-bind-button
style: primary
label: Insert Into Note
action:
  type: insertIntoNote
  line: "last" <-------- Not sure how to do this
  value: "Hello, world!"
  templater: false
```

**Comment by mProjectsCode** [comment]
not possible yet, see #296
